src/poetry/masonry/builders/editable.py

```diff
@@ -174,7 +174,11 @@ def _add_scripts(self) -> list[Path]:
         scripts = entry_points.get("console_scripts", [])
         for script in scripts:
             name, script = script.split(" = ")
-            module, callable_ = script.split(":")
+            try:
+                module, callable_ = script.split(":")
+            except ValueError as exc:
+                msg = f"{exc.args}  - Failed to parse script entry point '{script}'"
+                raise ValueError(msg) from exc
```

Comment:
```suggestion
                raise ValueError(f"Failed to parse script entry point '{script}'") from exc
```

Reply:
> Several lint tools recommend against interpolation when instantiating Exceptions.

Sorry, I don't follow. Can you give an example?

My reasoning is that you get the original exception via `from exc`. Thus, you don't need `exc.args` it in the derived exception.

Reply:
Poetry does not display the stack trace so you have to add the exception info to the message so the user can see what the symptom and cause is in one message

Reply:
Also the example
```
resources.py:45: [EM102] Exception must not use an f-string literal, assign to variable first
```

Reply:
Thanks for the example. I read the [reasoning to EM102](https://pypi.org/project/flake8-errmsg/0.4.0/) and I'm not sure I'm convinced (especially since we are not printing the traceback by default). Normally, we are using string literals and f-strings without assigning it to a variable first, so there is no reason to deviate from that here.

> Poetry does not display the stack trace so you have to add the exception info to the message so the user can see what the symptom and cause is in one message

You will see the traceback by running poetry with `-vvv`. When not in verbose mode, we shouldn't print the original error message since it is not very helpful anyway. If you think that `f"Failed to parse script entry point '{script}'"` is not enough information, then what about `f"Failed to parse script entry point '{script}', ':' not found"` or similar?

By the way, we have the same script parsing with the same issue in https://github.com/python-poetry/poetry/blob/161b19cb4e4686fc5a0a7925001534a87f6c4052/src/poetry/console/commands/run.py#L73

Reply:
This also catches cases where someone does not place any ":"
Are we overthinking this?

Reply:
Good point, my proposal is not accurate enough. However, I still stand by my initial statement.

IMO `('not enough values to unpack (expected 2, got 1)',)  - Failed to parse script entry point '...'` is not a good error message. Without any context `not enough values to unpack` does not really help. `Failed to parse script entry point '...'` should be good enough. If you want more details you have to run `poetry install -v`. That way you will get the `not enough values to unpack` with the line were it happened.